site/source/tutorials/005_form_widgets/demo/initial/biz-e-corp/src/widgets/WorkerForm.ts

```diff
@@ -0,0 +1 @@
+ 
```

Comment:
as noted by the CI, this file, though empty, is missing an EOL character. Also the space should be a tab. :)